src/main/java/org/assertj/core/error/ShouldHaveCause.java

```diff
@@ -31,6 +31,10 @@ public static ErrorMessageFactory shouldHaveCause(Throwable actualCause, Throwab
     return new ShouldHaveCause(actualCause, expectedCause);
   }
 
+  public static ErrorMessageFactory shouldHaveCause(Throwable actualCause) {
```

Comment:
Add a unit test in a new  test class `ShouldHaveCause_create_Test`, take `ShouldHaveRootCause_create_Test` as an example (sorry `ShouldHaveCause_create_Test` should have already existed!)